src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -91,10 +92,23 @@
     @Override
     public boolean start() throws Exception {
         final PlaceholderTask task = new PlaceholderTask(getContext(), step.getLabel(), run);
-        if (Queue.getInstance().schedule2(task, 0).getCreateItem() == null) {
+        Queue.Item taskInQueue = Queue.getInstance().schedule2(task, 0).getCreateItem();
+        if (taskInQueue == null) {
             // There can be no duplicates. But could be refused if a QueueDecisionHandler rejects it for some odd reason.
             throw new IllegalStateException("failed to schedule task");
```

Comment:
This isn't a change of behavior, though, just a change of syntax - I'm just keeping the `taskInQueue` around for later use, otherwise it's the same as it was before in re: the `IllegalStateException` - if `...getCreateItem()` is `null`, it'll throw.